crime-applications-adaptor/application/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/factory/PoundSterlingTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.laa.crime.applications.adaptor.factory;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.math.BigDecimal;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class PoundSterlingTest {
+  @ParameterizedTest
+  @MethodSource("pennyValues")
+  void shouldSuccessfullyConvertPenniesToPounds(Number pennies, BigDecimal expectedValue) {
+    BigDecimal actualValue = PoundSterling.ofPennies(pennies).toPounds();
+
+    assertEquals(actualValue, expectedValue);
```

Comment:
Can you switch the order of the parameters to match that of the method signature?

`assertEquals(Object expected, Object actual)`